pkg/arvo/lib/quiz.hoon

```diff
@@ -0,0 +1,57 @@
+|_  [eny=@uv runs=@ud]
+++  check  |=  [vax=vase]
```

Comment:
style: `|=` should go on the next line, indented once fromt the `++` line, like this:
```hoon
++  check
  |=  vax=vase
  =?  runs  =(0 runs)  100
  ...
```
Also, the sample does not need enclosing `[]`s.